auth/auth-router.js

```diff
@@ -1,11 +1,48 @@
 const router = require('express').Router();
+const bcrypt = require('bcryptjs')
+const jwt = require('jsonwebtoken')
 
-router.post('/register', (req, res) => {
-  // implement registration
+const User = require('./auth-model.js')
+
+router.post('/register', async (req, res) => {
+  const user = req.body
+  const hashedPW = bcrypt.hashSync(user.password, 10)
```

Comment:
Before you hash the password, you will want to make sure the user sent a username and password. If they don't you want to send back a 400 error.